ghost/values.yaml

```diff
@@ -1,33 +1,25 @@
-image:
-  repository: pennlabs/ghost-s3
-  tag: latest
-  pullPolicy: Always
+image_tag: latest
```

Comment:
Can we remove this? It'll just get overriden, so I think it's more confusing than worth it.

Reply:
Fixed